vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/ProjectFormLayout.java

```diff
@@ -74,6 +77,23 @@ public ProjectFormLayout() {
         .bind((ProjectInformation::getProjectObjective),
             ProjectInformation::setProjectObjective);
 
+    fundingField = new FundingField("Funding Information");
+    binder.forField(fundingField)
+        .withValidator(field -> {
```

Comment:
The element you are passing into the lambda is not the field but rather its value. So I suggest renaming it to `fundingInformation` or `funding`